src/Logging/LoggingEventHandler.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Utility
+{
+    internal class LoggingEventHandler
+    {
+        private HashSet<Action<string, string, Exception>> _eventHandlers = new HashSet<Action<string, string, Exception>>();
+
+        public void Subscribe(Action<string, string, Exception> handler)
+        {
+            _eventHandlers.Add(handler);
+        }
+
+        public void Unsubscribe(Action<string, string, Exception> handler)
```

Comment:
Doesn't seem to be used. Remove?

Reply:
I'll remove it for now - maybe worth discussing whether unsubscribing is ever necessary